packages/edit-post/src/components/layout/index.js

```diff
@@ -491,9 +491,8 @@ function Layout( {
 				hasBlockSelected: !! getBlockSelectionStart(),
 				showIconLabels: get( 'core', 'showIconLabels' ),
 				isDistractionFree: get( 'core', 'distractionFree' ),
-				showMetaBoxes:
-					( isNotDesignPostType && ! isZoomOut() ) ||
-					isDirectlyEditingPattern,
+				// @ZOOMOUTMODE
```

Comment:
For this one, I'm thinking of re-instating it, as it's kind of nice for it to get the metaboxes out of the way IMO.

https://github.com/user-attachments/assets/e9907606-7e22-45ed-a9f7-cf216eb920dd

Reply:
Re-instated this one.